src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -166,39 +160,58 @@
              [:dispatch [:wallet/flush-collectibles-fetched]])]})))
 
 (rf/reg-event-fx
- :wallet/get-collectible-details
- (fn [{:keys [db]} [collectible-id aspect-ratio]]
+ :wallet/navigate-to-collectible-details
+ (fn [{:keys [db]}
+      [{{collectible-id :id :as collectible} :collectible
+        aspect-ratio                         :aspect-ratio
+        gradient-color                       :gradient-color}]]
    (let [request-id               0
          collectible-id-converted (cske/transform-keys transforms/->PascalCaseKeyword collectible-id)
          data-type                (collectible-data-types :details)
          request-params           [request-id [collectible-id-converted] data-type]]
-     {:db (assoc-in db [:wallet :last-collectible-aspect-ratio] aspect-ratio)
+     {:db (assoc-in db
+           [:wallet :ui :collectible]
+           {:details        collectible
+            :aspect-ratio   aspect-ratio
+            :gradient-color gradient-color})
       :fx [[:json-rpc/call
             [{:method   "wallet_getCollectiblesByUniqueIDAsync"
               :params   request-params
               :on-error (fn [error]
                           (log/error "failed to request collectible"
-                                     {:event  :wallet/get-collectible-details
+                                     {:event  :wallet/navigate-to-collectible-details
                                       :error  error
-                                      :params request-params}))}]]]})))
+                                      :params request-params}))}]]
+           ;; We delay the navigation because we need re-frame to update the DB on time.
+           ;; By doing it, we skip a blink while visiting the collectible detail page.
+           [:dispatch-later
+            {:ms       1
```

Comment:
Any reason why `1ms` why chosen specifically? Or would the delay of 1ms be sufficient because we can't tell how long re-frame would take to finish the job?